updateengine/status.go

```diff
@@ -28,6 +31,56 @@ type Status struct {
 	NewSize          int64
 }
 
+var (
+	errStatusNil    = errors.New("dbus signal is nil")
+	errStatusLength = errors.New("signal body has insufficient length")
+	errStatusType   = errors.New("signal body has incorrect type")
+	statusZero      = Status{0, 0, "", "", 0}
```

Comment:
No need for this here. You can either move it to be local to the function, or even better just `return Status{}, errFoo`.

Reply:
Made an edit - how's this?